.github/workflows/reusable-docker-arm-build.yaml

```diff
@@ -61,15 +61,20 @@ jobs:
       - name: Set Version
         run: |
           echo "GIT_VERSION=$(git describe --always --tags --dirty)" >> $GITHUB_ENV
-      - name: Set Image name (master)
-        if: ${{ github.ref == 'refs/heads/master' }}
-        run: |
-          IMAGE_REF=$(SOFTWARE_VERSION=${{ inputs.k3s_version }} SOFTWARE_VERSION_PREFIX=k3s FLAVOR=${{ inputs.flavor }} FLAVOR_RELEASE="${{ inputs.flavor_release }}" MODEL=${{ inputs.model }} TARGETARCH=arm64 VARIANT=${{ inputs.variant }} REGISTRY_AND_ORG="${{ env.REGISTRY }}" RELEASE=master kairos-agent versioneer container-artifact-name)
-          echo "IMAGE_REF=${IMAGE_REF}" >> $GITHUB_ENV
-      - name: Set Image name (release/PR)
-        if: ${{ startsWith(github.ref, 'refs/tags/v') || github.event_name == 'pull_request' }}
+          if [ "$GITHUB_REF" == "refs/heads/master" ]; then
+            echo "RELEASE="master"" >> $GITHUB_ENV
+          else
+            echo "RELEASE=$GIT_VERSION" >> $GITHUB_ENV
+          fi
+
+      - name: Set Image name
         run: |
-          IMAGE_REF=$(SOFTWARE_VERSION=${{ inputs.k3s_version }} SOFTWARE_VERSION_PREFIX=k3s FLAVOR=${{ inputs.flavor }} FLAVOR_RELEASE="${{ inputs.flavor_release }}" MODEL=${{ inputs.model }} TARGETARCH=arm64 VARIANT=${{ inputs.variant }} REGISTRY_AND_ORG="${{ env.REGISTRY }}" RELEASE=${{ env.GIT_VERSION }} kairos-agent versioneer container-artifact-name)
+          SOFTWARE_VERSION="${{ inputs.k3s_version }}"
+          if [ -n "$SOFTWARE_VERSION" ]; then
+            SOFTWARE_VERSION="v$SOFTWARE_VERSION"
+          fi
+
+          IMAGE_REF=$(SOFTWARE_VERSION=${SOFTWARE_VERSION} SOFTWARE_VERSION_PREFIX=k3s FLAVOR=${{ inputs.flavor }} FLAVOR_RELEASE="${{ inputs.flavor_release }}" MODEL=${{ inputs.model }} TARGETARCH=arm64 VARIANT=${{ inputs.variant }} REGISTRY_AND_ORG="${{ env.REGISTRY }}" RELEASE=${{ env.RELEASE }} kairos-agent versioneer container-artifact-name)
```

Comment:
`RELEASE=${{ env.RELEASE }}` is redundant (env is already set) but better put it there to make it clear what value is being used?